Cargo.toml

```diff
@@ -16,44 +16,47 @@ resolver = "2"
 graceful-shutdown = { path = "src/utils/graceful-shutdown", version = "0.31.2", default-features = false }
 observability = { path = "src/utils/observability", version = "0.31.2", default-features = false }
 # Utils (core)
-container-runtime = { git = "https://github.com/kamu-data/kamu-cli", tag = "v0.195.1", version = "0.195.1", default-features = false }
-database-common = { git = "https://github.com/kamu-data/kamu-cli", tag = "v0.195.1", version = "0.195.1", default-features = false }
-database-common-macros = { git = "https://github.com/kamu-data/kamu-cli", tag = "v0.195.1", version = "0.195.1", default-features = false }
-http-common = { git = "https://github.com/kamu-data/kamu-cli", tag = "v0.195.1", version = "0.195.1", default-features = false }
-internal-error = { git = "https://github.com/kamu-data/kamu-cli", tag = "v0.195.1", version = "0.195.1", default-features = false }
-messaging-outbox = { git = "https://github.com/kamu-data/kamu-cli", tag = "v0.195.1", version = "0.195.1", default-features = false }
-random-names = { git = "https://github.com/kamu-data/kamu-cli", tag = "v0.195.1", version = "0.195.1", default-features = false }
-time-source = { git = "https://github.com/kamu-data/kamu-cli", tag = "v0.195.1", version = "0.195.1", default-features = false }
+container-runtime = { git = "https://github.com/kamu-data/kamu-cli", branch = "feature/727-Private-Datasets-Update-use-case-Creating-a-dataset", default-features = false }
```

Comment:
TODO: set the correct version when the PR is merged
- https://github.com/kamu-data/kamu-cli/pull/761

Reply:
Done